packages/core/src/components/Avatar/Avatar.tsx

```diff
@@ -3,7 +3,7 @@ import { theme } from "@hitachivantara/uikit-styles";
 import { AvatarProps as MuiAvatarProps } from "@mui/material/Avatar";
 import clsx from "clsx";
 import { CSSProperties } from "react";
-import { HvBaseProps } from "types";
+import { HvBaseProps } from "../../types";
 import { useImageLoaded } from "../../hooks";
```

Comment:
here the import should be from alias hooks

Reply:
PR updated.